src/combat/armor/armor-modifier.ts

```diff
@@ -0,0 +1,30 @@
+import { ItemSpawnEvent } from 'org.bukkit.event.entity';
+import { modifyArmor } from './armor-config';
+import { Material } from 'org.bukkit';
+import { PrepareItemCraftEvent } from 'org.bukkit.event.inventory';
+
+function isArmor(type: Material) {
+  const str = type.toString();
```

Comment:
Consider a Set of materials (now that they work) instead of multiple string comparisons.